docusaurus.config.js

```diff
@@ -6,7 +6,7 @@ const latestVersion = require("./src/versions").versionMappings[0].docsVersion;
 module.exports = {
   title: "Camunda 8 Docs",
   tagline:
-    "Orchestrate complex process flows, automate across people, systems, and devices, and transform your organization.",
+    "Get started with Camunda 8 SaaS or Self-Managed universal process orchestration.",
```

Comment:
🤔 I don't love this, but I can't figure out why. Is this copy you got from Marketing? 

> Get started with Camunda 8, the universal process orchestrator. 

> Get started with Camunda 8 SaaS or Self-Managed for universal process orchestration.

Reply:
This isn't from Marketing, but I tried to include the following important elements in it: Get Started (echoes the button), 2 main products/paths (SaaS and Self-Managed), and the "universal process orchestration" phrasing that is a focus currently. I think it also needs to be snappier, as the previous tagline was too long to scan quickly.

If we stick with the current style, I like your second option: "Get started with Camunda 8 Saas or Self-Managed for universal process orchestration."

We could also use alternative tagline styles such as:

* Standard documentation subtitle. e.g. "Explore Camunda 8 SaaS and Self-Managed documentation."
* Search focused (once we have a search bar). e.g. "Find answers to your technical questions and learn more about Camunda 8."

@christinaausley what do you think?

Reply:
How about "Get started with the Camunda 8 SaaS and Self-Managed documentation to orchestrate your process flows."?

Reply:
another alternative, but I have no stake in this:

Reply:
I like your alternative @christinaausley, just shortened for verb + description pattern, so I've changed it to:

"Start orchestrating your processes with Camunda 8 SaaS or Self-Managed."